pkg/webhook/common_test.go

```diff
@@ -0,0 +1,42 @@
+package webhook
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+)
+
+func TestCongifureWebhookServer(t *testing.T) {
+	expectedServer := &webhook.Server{
+		TLSMinVersion: "1.3",
+	}
+
+	if *clientCAName != "" {
+		expectedServer.ClientCAName = *clientCAName
+	}
+
+	tc := []struct {
+		Name           string
+		Server         *webhook.Server
+		ExpectedServer *webhook.Server
+	}{
+		{
+			Name:           "Wbhook server config",
+			Server:         &webhook.Server{},
+			ExpectedServer: expectedServer,
+		},
+	}
+
+	for _, tt := range tc {
+		t.Run(tt.Name, func(t *testing.T) {
+			server := congifureWebhookServer(tt.Server)
+			expectedServer.TLSOpts = server.TLSOpts
```

Comment:
I mean tests that use this via `net/http/httptest` and assert that it does what you expect.  You are enforcing a security boundary here.

Reply:
@enj Appologies for the misunderstanding! I added the test to assert the expected behavior.